src/m4/jtr_x86_logic.m4

```diff
@@ -481,4 +485,9 @@ fi
 
 CC="$CC_BACKUP"
 CFLAGS="$CFLAGS_BACKUP"
+
+if test "x$simd" = xno ; then
+CPU_STR="SIMD_disabled"
+HT="-UHAVE_HT"
```

Comment:
Here's another thing I told you NOT to do. Why on earth would we want to disable HT for --disable-simd? Hyper-threading is not SIMD.

Reply:
BTW we're not actually disabling HT, just hiding the test from any code that wants to know. Current code probably won't be affected at all but still, this is just wrong.

Reply:
Ok, I will remove the HT= changes.